lightkurve/targetpixelfile.py

```diff
@@ -80,6 +82,107 @@ def __getitem__(self, key):
             copy[1].data = copy[1].data[selected_idx]
         return self.__class__(copy, quality_bitmask=self.quality_bitmask, targetid=self.targetid)
 
+    def __len__(self):
+        return len(self.time)
+
+
+    def _verify_other_(self, other):
```

Comment:
Even though it's a helper function, let's have a one-line docstring here to explain what this function is for.